mozdownload/scraper.py

```diff
@@ -639,7 +639,9 @@ def get_build_info(self):
             self.build_index = 0
             self.logger.info('Selected build: build%s' % self.build_number)
         else:
-            self.logger.info('Selected build: build%s' % self.build_index)
+            # Otherwise displays incorrect build number
```

Comment:
Not sure what I should get from this comment. :) I think we can remove it given that your fix is below.